RELEASE_NOTES.rst

```diff
@@ -13,6 +13,7 @@ Migration notes
 All changes
 -----------
 
+- Correct check for unscaled-infeasibilities to check for `solvestat=4` (:pull:`676`).
```

Comment:
Because of point (a) in my review, I would rather we remove this line and just update the one existing line below to read like this:
```
- Add additional oscillation detection mechanism for MESSAGE-MACRO iterations; attempt to use `barcrossalg=2` (dual crossover) when MESSAGE solves with unscaled infeasibilities (:pull:`645`, :pull:`676`).
```
The reason: from the point of view of a user using only the stable releases, there was never a release where there was a check for unscaled infeasibilities using something other than `solvestat=4`. So, there is nothing to be "Correct"ed (per the added line). Compared to v3.6.0, this is a new feature that will appear in v3.7.0 as the result of these two PRs.

Reply:
done!